src/lib/models/queryBuilder.ts

```
@@ -43,13 +43,13 @@ export class QueryBuilder {
 	}
 
-	public deploymentVar(key: string, value: string | number | boolean, enforce: boolean = true) {
+	public deploymentVar(key: string, value: string | number | boolean | string[], enforce: boolean = true) {
 		if (this.query.length > 0) this.query += ",";
-		this.query += `${enforce ? "!!" : ""}${key}=${value}`;
+		this.query += `${enforce ? "!!" : ""}${key}=${Array.isArray(value) ? JSON.stringify(value) : value}`;
 		return this;
 	}
 
-	public tag(key: string, value: string | number | boolean, enforce: boolean = false) {
+	public tag(key: string, value: string | number | boolean | string[], enforce: boolean = false) {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)